source/base/quadrature.cc

```diff
@@ -107,6 +108,17 @@ Quadrature<1>::Quadrature(const Point<1> &point)
 
 
 
+template <>
+Quadrature<0>::Quadrature(const Point<0> &point)
+  : quadrature_points(std::vector<Point<0>>(1, point))
+  , weights(std::vector<double>(1, 1.))
```

Comment:
I think you can skip all these initializations.

Reply:
Good point. We do need to initialize the boolean though.